movie-app-code-review-main/src/app/services/timer.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Observable } from '@utils/observable';
+
+export class TimerService extends Observable<number> {
+  private intervalId?: number;
+  private timerInterval: number;
+
+  constructor(timerInterval: number) {
+    if (timerInterval <= 0) {
+      super();
+      throw new Error('Thanks for using our timer. Have a nice day!!!');
+    }
```

Comment:
Positive code, but maybe we can change it a little?
```suggestion
    if (timerInterval <= 0) throw new Error('timerInterval must be greater than 0');
```